data/nav.yml

```diff
@@ -429,8 +429,33 @@
           path: "apis/rest-api"
         - name: "Limits"
           path: "apis/rest-api/limits"
-        - name: "Pipelines REST API"
+        - name: "Agent"
+          children:
+            - name: "Overview"
+              path: "apis/agent-api"
+            - name: "Metrics"
+              path: "apis/agent-api/metrics"
+        - name: "Analytics"
           children:
+            - name: "Flaky tests"
+              path: "apis/rest-api/analytics/flaky-tests"
+            - name: "Runs"
+              path: "apis/rest-api/analytics/runs"
+            - name: "Suites"
+              path: "apis/rest-api/analytics/suites"
+            - name: "Tests"
+              path: "apis/rest-api/analytics/tests"
+        - name: "Organizations"
+          children:
+            - name: "Overview"
+              path: "apis/rest-api/organizations"
+            - name: "Members"
+              path: "apis/rest-api/organizations/members"
+              pill: "beta"
+        - name: "Pipelines "
+          children:
+            - name: "Overview"
+              path: "apis/rest-api/pipelines"
```

Comment:
I needed to add a space after the word Pipelines here (i.e. `"Pipelines "`) to prevent the **Pipelines** section in the 'global' nav from being highlighted too when viewing its Overview page.

This seems to be a bit of a bug in our documentation site, whereby this kind of item structure within this YAML file:

```
        - name: "Something specific"
          ...
            - name: "Something else"
              ...
```

Would could cause any element that's also called "Something specific" elsewhere on the page to become highlighted too.

Here's an example of what I mean:

![multiple-elements-being-highlighted](https://github.com/buildkite/docs/assets/23667807/d76affe4-97a5-4c70-82ed-7f580ed5597e)

@dannymidnight , would you know if there's anyway to fix this issue with our docs site?